### DISCUSSION TITLE
Video feed not working in QGC

### DISCUSSION THREAD

**Discussion by estevezb** [RosettaDrone/rosettadrone#178] [General]
Hi, would be interested in knowing how you'll got the video feed from Rosetta onto your QGC. Running QGround control on my laptop entered my local IPv4 Address. Oddly it is able to show a connection -I do see the green circles for DJI and GCS in the Rosetta app but the video feed says 'Waiting for video' in QGC. Can you share your settings or screen shots for inputs into Rosetta app settings? My settings and some screen shots, what am i doing wrong here (also tried changing port number in the 'Drone configuration' menu for the 'Use External H.264 input' to Port number 5600. That did not solve it for me either. (FYI Using the Air2S). 
![image](https://github.com/RosettaDrone/rosettadrone/assets/56987765/997ff8f6-b94b-4952-9d5d-70e907c9f01d)
![image](https://github.com/RosettaDrone/rosettadrone/assets/56987765/71c8829b-ab2b-4870-9a92-e4d90e5aea3c)
![image](https://github.com/RosettaDrone/rosettadrone/assets/56987765/6a6dfa9f-fc7a-4106-8b47-194cef433624).

**Comment by Reto80** [comment]
Ich habe das selbe Problem, kontest Du es mittlerweile fixen??? Und wenn ja wie?? Arbeite mit einer DJI Mavic Air 1Gen.. Thank You for theAwnser🤪🫣🤔😉
